eth/handler.go

```diff
@@ -151,19 +154,20 @@ func newHandler(config *handlerConfig) (*handler, error) {
 		config.EventMux = new(event.TypeMux) // Nicety initialization for tests
 	}
 	h := &handler{
-		nodeID:         config.NodeID,
-		networkID:      config.Network,
-		eventMux:       config.EventMux,
-		database:       config.Database,
-		txpool:         config.TxPool,
-		noTxGossip:     config.NoTxGossip,
-		chain:          config.Chain,
-		peers:          newPeerSet(),
-		txBroadcastKey: newBroadcastChoiceKey(),
-		requiredBlocks: config.RequiredBlocks,
-		quitSync:       make(chan struct{}),
-		handlerDoneCh:  make(chan struct{}),
-		handlerStartCh: make(chan struct{}),
+		nodeID:              config.NodeID,
+		networkID:           config.Network,
+		eventMux:            config.EventMux,
+		database:            config.Database,
+		txpool:              config.TxPool,
+		noTxGossip:          config.NoTxGossip,
+		txGossipNetRestrict: config.TxGossipNetRestrict,
+		chain:               config.Chain,
+		peers:               newPeerSet(),
+		txBroadcastKey:      newBroadcastChoiceKey(),
+		requiredBlocks:      config.RequiredBlocks,
+		quitSync:            make(chan struct{}),
+		handlerDoneCh:       make(chan struct{}),
+		handlerStartCh:      make(chan struct{}),
```

Comment:
```suggestion
		chain:               config.Chain,
		peers:               newPeerSet(),
		txBroadcastKey:      newBroadcastChoiceKey(),
		requiredBlocks:      config.RequiredBlocks,
		quitSync:            make(chan struct{}),
		handlerDoneCh:       make(chan struct{}),
		handlerStartCh:      make(chan struct{}),
		
		// OP Stack additions
		noTxGossip:          config.NoTxGossip,
		txGossipNetRestrict: config.TxGossipNetRestrict,
```
same here, if we add to the end this way, we reduce the upstream diff